### ISSUE TITLE
Perl unit tests: Cannot connect to service pfconfig!

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#5511]
**Describe the bug**
Since merge of #5348, #5507 and #5488, preliminary steps to run unit tests are failing on Debian and CentOS with following error message:

```
Cannot connect to service pfconfig! at /usr/local/pf/lib/pfconfig/cached.pm line 208.
```

[Example pipeline ](https://gitlab.com/inverse-inc/packetfence/-/jobs/578313781#L1036)

Error occured when we run https://github.com/inverse-inc/packetfence/blob/3c7d3b8abdb2bd3c8724c10cb5bc8579745674b7/t/db/setup_test_db.pl

**Comment by nqb** [comment]
I see some changes related to `pfconfig` in #5507

**Comment by jrouzierinverse** [comment]
Fixed by 350a9a1
